spikeinterface/core/numpyextractors.py

```diff
@@ -235,7 +235,7 @@ def __init__(self, units_dict):
             assert np.all(np.diff(times) >= 0), 'unsorted times'
         self._units_dict = units_dict
 
-    def get_unit_spike_train(self, unit_id, start_frame, end_frame):
+    def get_unit_spike_train(self, unit_id, start_frame = None, end_frame = None):
```

Comment:
By design because BaseSortingSegment is internal only.
I put args only everywhere to avoid mistake.
In short, you have to think about the start and end args when designing something.

This is nomally never call by the end user. So kwargs is useless.

In short I would prefer not.

Reply:
Ok, I'll change it then!